test/polygon/asset/assetERC1155Tunnels.test.ts

```diff
@@ -549,6 +554,65 @@ describe('Asset_ERC1155_Tunnels', function () {
       expect(balanceL2Tunnel).to.be.equal(0);
     });
 
+    it('transfer a single asset from L1 to L2', async function () {
+      const {
+        deployer,
+        AssetERC1155,
+        MockAssetERC1155Tunnel,
+        users,
+        mintAssetOnL1,
+      } = await setupAssetERC1155Tunnels();
+
+      const supplies = [20];
+
+      // Note: tokenIds cannot be 0, 1, 2, 3... as will revert with 'ID_TAKEN'
+      // IDs on L1 must follow the precise format as generated on L2
+      const tokenIds = [
+        '0x2de2299db048a9e3b8d1934b8dae11b8041cc4fd000000008000000000800800',
+      ];
+
+      await deployer.MockAssetERC1155Tunnel.setTransferLimit(
+        BigNumber.from('1')
+      );
+
+      const testMetadataHashArray = [];
+      testMetadataHashArray.push(
+        ethers.utils.formatBytes32String('metadataHash')
+      );
+      await mintAssetOnL1(users[0].address, tokenIds[0], supplies[0]);
+
+      // Transfer to L1 Tunnel
+      await AssetERC1155.connect(
+        ethers.provider.getSigner(users[0].address)
+      ).setApprovalForAll(MockAssetERC1155Tunnel.address, true);
+
+      await waitFor(
+        MockAssetERC1155Tunnel.connect(
+          ethers.provider.getSigner(users[0].address)
+        ).batchDepositToChild(users[0].address, tokenIds, supplies)
+      );
+
+      /*       const balanceUserL1 = await AssetERC1155['balanceOf(address,uint256)'](
```

Comment:
Is this commented for any particular reason?

Reply:
no, fixing it